src/Resources/ListingsItemsResource.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Jasara\AmznSPA\Resources;
+
+use Illuminate\Support\Arr;
+use Jasara\AmznSPA\AmznSPAHttp;
+use Jasara\AmznSPA\Constants\AmazonEnums;
+use Jasara\AmznSPA\Constants\MarketplacesList;
+use Jasara\AmznSPA\Contracts\ResourceContract;
+use Jasara\AmznSPA\DataTransferObjects\Requests\ListingsItems\ListingsItemPatchRequest;
+use Jasara\AmznSPA\DataTransferObjects\Requests\ListingsItems\ListingsItemPutRequest;
+use Jasara\AmznSPA\DataTransferObjects\Responses\ListingsItems\GetListingsItemResponse;
+use Jasara\AmznSPA\DataTransferObjects\Responses\ListingsItems\ListingsItemSubmissionResponse;
+use Jasara\AmznSPA\Traits\ValidatesParameters;
+
+class ListingsItemsResource implements ResourceContract
+{
+    use ValidatesParameters;
+
+    public const BASE_PATH = '/listings/2021-08-01/';
+
+    public function __construct(
+        private AmznSPAHttp $http,
+        private string $endpoint,
+    ) {
+    }
+
+    public function getListingsItem(
+        string $seller_id,
+        string $sku,
+        array $marketplace_ids,
+        ?string $issue_locale = null,
+        ?array $included_data = null,
+    ): GetListingsItemResponse {
+        $this->validateIsArrayOfStrings($marketplace_ids, MarketplacesList::allIdentifiers());
+        if ($included_data) {
+            $this->validateIsArrayOfStrings($included_data, AmazonEnums::INCLUDED_DATA);
+        }
+
+        $response = $this->http->get($this->endpoint . self::BASE_PATH . 'items/' . $seller_id . $sku, array_filter([
+            'SellerIs' => $seller_id,
```

Comment:
SellerIs -> SellerId

Reply:
This is in all the endpoints

Reply:
Actually I had a look at the docs to make, and it looks like you don't need to pass this in as data at all as you are already passing it in the path. Same with the SKU.